src/braket/default_simulator/openqasm/circuit.py

```diff
@@ -60,6 +61,12 @@ def add_instruction(self, instruction: [GateOperation, KrausOperation]) -> None:
         self.instructions.append(instruction)
         self.qubit_set |= set(instruction.targets)
 
+    def add_measure(self, target: int | list[int]):
+        if isinstance(target, list):
+            self.measured_qubits += target
+        else:
+            self.measured_qubits.append(target)
```

Comment:
for now, a set might be sufficient here. Eventually a set won't be enough information for mid-circuit measurements so this might be better